podman_compose.py

```diff
@@ -2606,7 +2606,7 @@ def get_excluded(compose, args):
     if args.services:
         excluded = set(compose.services)
         for service in args.services:
-            if not args.no_deps:
+            if service in compose.services and not getattr(args, "no_deps", False):
```

Comment:
Any reason to change `args.no_deps` to use getattr?

Reply:
This is one of the reasons I kept this PR in "work in progress", there were some cases (which I don't know if are valid or not) that `args.no_deps` was undefined.

I'm sorry, but I've been drowning in work lately. I plan to get back to podman-compose and try to help with its development, but it will still take a few weeks.

Reply:
Oh I see - compose_up_parse configures `no-deps` argument, but compose_down_parse doesn't.

Reply:
Maybe it makes sense to add a comment saying that `compose_down_parse` does not configure `no-deps` command argument. This would make it clear why the code is how it is.